### PR TITLE
Add a link to the GitHub repo

### PR DESCRIPTION
**Pull request by omarudolley**
Add a link to the GitHub repo from the app itself.  `pnpm install` updated the lock file.

latest_link

### CODE REVIEW

File: frontend/src/index.css [hunk 1]
```diff
@@ -78,3 +78,20 @@ body,
...
 }
+
+.nav {
+  width: 100%;
+  max-width: 40rem;
+  display: flex;
+  justify-content: end;
+}
+
+.nav-link {
```
Comment (meandrewandyou) [inline frontend/src/index.css:89]: Does this styles affects anything (`.nav-link`)?

**Review by meandrewandyou — APPROVED** [review]
Asked one question.

File: frontend/src/index.css (hunk 1 above)
Comment (omarudolley) [reply frontend/src/index.css:89]: with Screenshot 2024-04-12 110717 vs without 
Screenshot 2024-04-12 110751

**Review by TheHelias — APPROVED** [review]

**Review by fbjorn — APPROVED** [review]

File: frontend/src/assets/github.svg
Comment (lietu) [inline frontend/src/assets/github.svg:1]: Has anyone actually looked with their eyeballs at this image?

**Review by lietu — CHANGES_REQUESTED** [review]
Replace the GitHub image with a proper one

File: frontend/src/assets/github.svg
Comment (omarudolley) [reply frontend/src/assets/github.svg:1]: We should update this also https://github.com/ioxiocom/ioxio-docs/blob/main/src/lib/images/github.svg

Comment (lietu) [reply frontend/src/assets/github.svg:1]: Yes, please do

File: frontend/src/components/Navbar.jsx [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+export default function Navbar() {
+  return (
+    <div className="nav">
+      <a
+        className="nav-link"
+        href="https://github.com/ioxio-dataspace/example-app"
+        rel="noreferrer"
+        target="_blank"
+      >
+        <img className="nav-logo" src={github} alt="View source in GitHub" />
```
Comment (lietu) [inline frontend/src/components/Navbar.jsx]:
```suggestion
import GitHubLogo from "../assets/github.svg"

export default function Navbar() {
  return (
    <div className="nav">
      <a
        className="nav-link"
        href="https://github.com/ioxio-dataspace/example-app"
        rel="noreferrer"
        target="_blank"
      >
        <img className="nav-logo" src={GitHubLogo} alt="View source in GitHub" />
```

**Review by lietu — APPROVED** [review]

**Pull request merged**